pallets/frequency-tx-payment/Cargo.toml

```diff
@@ -35,30 +35,38 @@ common-runtime = { path = "../../runtime/common", default-features = false }
 pallet-balances = { workspace = true }
 pallet-utility = { workspace = true }
 
-# Frequency related dev dependencies
-pallet-msa = { path = '../msa' }
-
 
 [features]
 default = ["std"]
 runtime-benchmarks = [
   "frame-benchmarking/runtime-benchmarks",
+  "frame-support/runtime-benchmarks",
```

Comment:
@aramikm added these lines to allow pallet to be built, tested & debugged from inside the pallet directory